src/table/table.stories.ts

```diff
@@ -25,6 +27,50 @@ import {
 
 import { clone } from "../utils/utils";
 
+
+@Component({
+	selector: "app-table",
+	template: `
+		<ibm-table
+			[model]="model"
+			[size]="size"
+			[showSelectionColumn]="showSelectionColumn"
+			[striped]="striped"
+			(sort)="simpleSort($event)">
+		</ibm-table>
+	`
+})
+class TableStory implements OnInit, OnChanges {
+	@Input() model = new TableModel();
+	@Input() size = "md";
+	@Input() showSelectionColumn = true;
+	@Input() striped = true;
+	@Input() sortable = true;
+	@Input() simpleSort;
+
+	ngOnInit() {
+		this.model.header = [
+			new TableHeaderItem({
+				data: "Name",
+				sortable: this.sortable
+			}),
+			new TableHeaderItem({
+				data: "hwer",
+				style: {"width": "auto"},
+				sortable: this.sortable
+			})
+		];
+	}
+
+	ngOnChanges(changes: SimpleChanges): void {
+		if (changes.sortable) {
+			for (let column of this.model.header) {
```

Comment:
Ideally, making the whole table sortable/non-sortable would be done by providing/not providing sort function.

Can be checked for with `this.sort.observers.length > 0`

This also requires changes in template where `columns.sortable` is checked.

Reply:
You're right that we need a global flag as to whether the table can be sorted at all or not. The idea of the `TableHeaderItem.sortable` is to disable sorting for some of the columns while leaving it for others.

Reply:
We don't need to supply a special global flag, per se, presence of the sort observer (whether or not user said `(sort)="mySortingFunction()"` is a clear indicator that user wants to sort, so we can use that instead as `this.sort.observers.length > 0`

Reply:
I added changes to use `this.sort.observers.length > 0` as a global sorting flag and then tie it further down with the `column.sortable`.
I couldn't make the story nicer because I don't know how to set a conditional `(sort)` and that's why I left the `column.sortable` there.